### PR TITLE
fix(orb-ui): fix agent provisioning command

### LINKED ISSUE #418: orb-agent command in UI after agent provisioning
### Replace `mock` interface with host network interface (e.g. `eth0`). [See here](https://getorb.io/docs/#running-orb-agent) for more advanced options.
```
docker run -d --net=host
-e ORB_CLOUD_ADDRESS=<HOST>
-e ORB_CLOUD_MQTT_ID=<AGENTID>
-e ORB_CLOUD_MQTT_CHANNEL_ID=<CHANNELID>
-e ORB_CLOUD_MQTT_KEY=<AGENTKEY>
-e PKTVISOR_PCAP_IFACE_DEFAULT=mock
ns1labs/orb-agent
```

### PR DESCRIPTION
**Pull request by gpazuch**
closes #418 
Fix agent provisioning command

### CODE REVIEW

File: ui/src/app/pages/fleet/agents/key/agent.key.component.ts [hunk 1]
```diff
@@ -23,19 +24,20 @@ export class AgentKeyComponent implements OnInit {
...
 
   makeCommand2Copy() {
-    this.command2copy = `docker run --rm --net=host \\
-      -e ORB_CLOUD_ADDRESS=${document.location.protocol}//${document.location.hostname}/ \\
-      -e ORB_CLOUD_MQTT_ID='${this.agent.id}' \\
-      -e ORB_CLOUD_MQTT_CHANNEL_ID='${this.agent.channel_id}' \\
-      -e ORB_CLOUD_MQTT_KEY='${this.agent.key}' \\
-      -e PKTVISOR_PCAP_IFACE_DEFAULT=[ETH-INTERFACE] \\
-      ns1labs/orb-agent run`;
+    this.command2copy = `docker run --rm --net=host \
```
Comment (weyrick) [inline ui/src/app/pages/fleet/agents/key/agent.key.component.ts:34]: this needs to add `-d` as per https://github.com/ns1labs/orb/issues/418

File: ui/src/app/pages/fleet/agents/key/agent.key.component.ts [hunk 2]
```diff
@@ -23,19 +24,20 @@ export class AgentKeyComponent implements OnInit {
...
+-e ORB_CLOUD_ADDRESS=${ document.location.hostname } \
+-e ORB_CLOUD_MQTT_ID='${ this.agent.id }' \
+-e ORB_CLOUD_MQTT_CHANNEL_ID='${ this.agent.channel_id }' \
+-e ORB_CLOUD_MQTT_KEY='${ this.agent.key }' \
+-e PKTVISOR_PCAP_IFACE_DEFAULT=\$\{PKTVISOR_IFACE\} \
```
Comment (weyrick) [inline ui/src/app/pages/fleet/agents/key/agent.key.component.ts:39]: what is `\$\{PKTVISOR_IFACE\}` ?

File: ui/src/app/pages/fleet/agents/key/agent.key.component.ts [hunk 3]
```diff
@@ -23,19 +24,20 @@ export class AgentKeyComponent implements OnInit {
...
-    this.command2copy = `docker run --rm --net=host \\
-      -e ORB_CLOUD_ADDRESS=${document.location.protocol}//${document.location.hostname}/ \\
-      -e ORB_CLOUD_MQTT_ID='${this.agent.id}' \\
-      -e ORB_CLOUD_MQTT_CHANNEL_ID='${this.agent.channel_id}' \\
-      -e ORB_CLOUD_MQTT_KEY='${this.agent.key}' \\
-      -e PKTVISOR_PCAP_IFACE_DEFAULT=[ETH-INTERFACE] \\
-      ns1labs/orb-agent run`;
+    this.command2copy = `docker run --rm --net=host \
+-e ORB_CLOUD_ADDRESS=${ document.location.hostname } \
+-e ORB_CLOUD_MQTT_ID='${ this.agent.id }' \
```
Comment (weyrick) [inline ui/src/app/pages/fleet/agents/key/agent.key.component.ts:36]: let's get rid of single quotes here and for channel and key

**Review by weyrick — CHANGES_REQUESTED** [review]

**Review by gpazuch — COMMENTED** [review]
@weyrick command should be ok now

**Review by weyrick — APPROVED** [review]

**Pull request merged**